test/anvil/anvil-global-setup.ts

```diff
@@ -1,24 +1,71 @@
 import { CreateAnvilOptions, createAnvil } from '@viem/anvil';
 import { sleep } from '../lib/utils/promises';
 
+type NetworkSetup = {
+    rpcEnv: string;
+    fallBackRpc: string | undefined;
+    port: number;
+    forkBlockNumber: bigint;
+};
+
+type Networks = NetworkSetup[];
+
+const NETWORKS: Networks = [
+    {
+        rpcEnv: 'ETHEREUM_RPC_URL',
+        fallBackRpc: 'https://cloudflare-eth.com',
+        port: 8545,
+        forkBlockNumber: 18043296n,
+    },
+    {
+        rpcEnv: 'POLYGON_RPC_URL',
+        // Public Polygon RPCs are usually unreliable
+        fallBackRpc: undefined,
+        port: 8137,
+        // Note - this has to be >= highest blockNo used in tests
+        forkBlockNumber: 44215395n,
+    },
+        {
+        rpcEnv: 'FANTOM_RPC_URL',
+        // Public Fantom RPCs are usually unreliable
+        fallBackRpc: undefined,
+        port: 8138,
+        forkBlockNumber: 65313450n,
+    },
+];
+
 export default async function () {
+    for (const network of NETWORKS) {
+        const config = getAnvilOptions(network);
+        await startFork(config);
+    }
+}
+
+function getAnvilOptions(network: NetworkSetup): CreateAnvilOptions {
     let forkUrl: string;
-    if (process.env.ETHEREUM_RPC_URL) {
-        forkUrl = process.env.ETHEREUM_RPC_URL;
+    if (process.env[network.rpcEnv]) {
```

Comment:
This "if" statement works if the variable is available on the .env file, but is empty, but does not work if the variable is not available in the .env file at all(process.env[network.rpcEnv] is a string 'undefined' in this case), so it should be`process.env[network.rpcEnv] !== 'undefined' && process.env[network.rpcEnv]`

Reply:
Thanks for pointing that out!